packages/react-admin/src/Admin.js

```diff
@@ -29,10 +29,10 @@ const Admin = ({
     dashboard,
     history,
     locale,
-    messages = {},
     menu = Menu,
     catchAll,
     dataProvider,
+    i18nProvider,
```

Comment:
Can we set a default `i18nProvider` which would load messages as before ?

Reply:
```locale => this.props.messages[locale]``` ?